modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/servlet/filter/PortalMonitoringControlImpl.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.monitoring.internal.servlet.filter;
+
+import com.liferay.portal.kernel.monitoring.PortalMonitoringControl;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Joao Victor Alves Silva
+ */
+@Component(enabled = false, service = PortalMonitoringControl.class)
+public class PortalMonitoringControlImpl implements PortalMonitoringControl {
```

Comment:
@MrJohn1911 Any reason to use inner classes in the other 2 cases, while making this one a separate component?

Reply:
Hi @dantewang, in this case, I made it in a separate component because the new component don't share any field or method with the other component, but if you think it's better making it an inner class, I can resend this pull.